.github/workflows/webpack-jumpstart.yml

```diff
@@ -11,16 +11,21 @@ jobs:
       pull-requests: none
     timeout-minutes: 20
     env:
-      MERGE_SHA: ${{ github.event.pull_request.merge_commit_sha || github.sha }}
+      BASE_SHA: ${{ github.sha }}
+      HEAD_SHA: ${{ github.event.pull_request.head.sha || github.sha }}
     steps:
       - name: Build jumpstart tarball
         run: |
           id="$(docker container create -u node -w /home/node node /bin/sh -c \
-            'git init cockpit &&
+            "set -x &&
```

Comment:
.. I was missing that all of them are chained with `&&`, so that's not important. (Again stylistics: `set -e` and a multi-line string would look a bit cleaner?)

Reply:
interesting proposal.  I'll give it a try :)

Reply:
I moved to your idea and added an `origin` remote to shorten the long line.